src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/helpers/satetyConcernHelper.ts

```diff
@@ -78,3 +55,32 @@ export const SafetyConcernsHelper = (userCase, keys, sessionKey, childField, typ
   }
   return '';
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function getChildBodyHtml(typeOfUser: any, bodyHtml: any, keys: any, FoundElement: any, userCase: any) {
+  if (typeOfUser === 'child') {
+    bodyHtml += HTML.H4 + keys['childrenConcernedAboutLabel'] + HTML.H4_CLOSE;
+    if (FoundElement.hasOwnProperty('childrenConcernedAbout')) {
+      bodyHtml += HTML.UNORDER_LIST;
+      if (
+        Array.isArray(FoundElement['childrenConcernedAbout']) &&
+        FoundElement['childrenConcernedAbout'][0] === 'All the children in application'
```

Comment:
How this is going to work when it is in welsh? And please use enums when referring constants.

Reply:
Here that 'All the children in application' is taken as a value for the field and it works same like yesorno.yes , and when coming to the label ... it changes accordingly with the welsh or english.